core/src/main/resources/application.properties

```diff
@@ -60,6 +60,8 @@ yona.security.passwordLength = 32
 yona.security.dosProtectionEnabled = false
 yona.security.dosProtectionWindow = PT5M
```

Comment:
See my comment above on ``updateUser``. I question there whether it's sufficient to have a single window setting.